src/context/walletContext/walletContext.tsx

```diff
@@ -28,18 +26,12 @@ interface WalletContextType {
     walletColor: string
 }
 
-function isPeanut(wallet: interfaces.IWallet | undefined) {
+function isPeanut(wallet: interfaces.IDBWallet | undefined) {
```

Comment:
thought: wdyt of maintaining an interface in FE and also maintaining the backend implementation? Seems uneccesarily duplicate

Reply:
Hmmm, it is duplicate and prone to errors. One way of tackling this is publishing a types package from the backend and use that in the front. That is less error prone but also less flexible. Not sure about using the sdk for this